imgix/urlhelper.py

```diff
@@ -14,6 +14,42 @@
 
 
 class UrlHelper(object):
+    """
+    Helper class to create single domain Imgix URLs. Example:
+
+      >>> str(UrlHelper('demos.imgix.net', '/bridge.png', opts={'w': 100}))
+      https://demos.imgix.net/bridge.png?w=100
+
+    Parameters
+    ----------
+    domain : str
+    path : str
+    scheme : { 'https', 'http' }
+    sign_key : str or None
+        Use the key to generate signed image URLs. URLs are not signed by
+        default. (default `None`)
+    sign_mode : {`SIGNATURE_MODE_QUERY`, `SIGNATURE_MODE_PATH`}
+        If `SIGNATURE_MODE_QUERY`, sign the whole URL. `SIGNATURE_MODE_PATH`
+        not supported yet. (default `SIGNATURE_MODE_QUERY`)
+    sign_with_library_version : bool
+        If `True`, each created URL is suffixed with 'ixlib' parameter
+        indicating the library used for generating the URLs. (default `True`)
+    opts : dict
```

Comment:
Outside the scope of this PR, but we should rename this to `params`.

Reply:
I agree. Will file a Monday task.